integrationTests/vm/txsFee/utils/utils.go

```diff
@@ -0,0 +1,138 @@
+
+	retCode, err := testContext.TxProcessor.ProcessTransaction(tx)
+	require.Equal(t, vmcommon.Ok, retCode)
+	require.Nil(t, err)
+	require.Nil(t, testContext.GetLatestError())
+
+	_, err = testContext.Accounts.Commit()
+	require.Nil(t, err)
+
+	expectedBalance := big.NewInt(89030)
+	vm.TestAccount(t, testContext.Accounts, owner, senderNonce+1, expectedBalance)
+
+	// check accumulated fees
+	accumulatedFee := testContext.TxFeeHandler.GetAccumulatedFees()
+	require.Equal(t, big.NewInt(10970), accumulatedFee)
+
+	scAddr, _ = testContext.BlockchainHook.NewAddress(owner, 0, factory.ArwenVirtualMachine)
+
+	developerFees := testContext.TxFeeHandler.GetDeveloperFees()
+	require.Equal(t, big.NewInt(368), developerFees)
+
+	return scAddr, owner
+}
+
+// PrepareRelayerTxData -
+func PrepareRelayerTxData(innerTx *transaction.Transaction) []byte {
+	userTxBytes, _ := protoMarshalizer.Marshal(innerTx)
+	return []byte(core.RelayedTransaction + "@" + hex.EncodeToString(userTxBytes))
+}
+
+// CheckOwnerAddr -
+func CheckOwnerAddr(t *testing.T, testContext *vm.VMTestContext, scAddr []byte, owner []byte) {
+	acc, err := testContext.Accounts.GetExistingAccount(scAddr)
+	require.Nil(t, err)
+
+	userAcc, ok := acc.(state.UserAccountHandler)
+	require.True(t, ok)
+
+	currentOwner := userAcc.GetOwnerAddress()
+	require.Equal(t, owner, currentOwner)
+}
+
+// TestAccount -
+func TestAccount(
+	t *testing.T,
+	accnts state.AccountsAdapter,
+	senderAddressBytes []byte,
+	expectedNonce uint64,
+	expectedBalance *big.Int,
+) *big.Int {
+
+	senderRecovAccount, err := accnts.GetExistingAccount(senderAddressBytes)
+	if err != nil {
+		assert.Nil(t, err)
+		return big.NewInt(0)
+	}
+
+	senderRecovShardAccount := senderRecovAccount.(state.UserAccountHandler)
+
+	require.Equal(t, expectedNonce, senderRecovShardAccount.GetNonce())
+	require.Equal(t, expectedBalance, senderRecovShardAccount.GetBalance())
+	return senderRecovShardAccount.GetBalance()
+}
+
+// ProcessSCRResult -
+func ProcessSCRResult(
+	t *testing.T,
+	testContext *vm.VMTestContext,
+	tx data.TransactionHandler,
+	expectedCode vmcommon.ReturnCode,
+	expectedErr error,
+) {
+	scProcessor := testContext.ScProcessor
+	scrProcessor, ok := scProcessor.(process.SmartContractResultProcessor)
+	require.True(t, ok)
+
+	scr, ok := tx.(*smartContractResult.SmartContractResult)
+	require.True(t, ok)
+
+	retCode, err := scrProcessor.ProcessSmartContractResult(scr)
+	require.Equal(t, expectedCode, retCode)
+	require.Equal(t, expectedErr, err)
+}
+
+// GetIntermediateTransactions -
+func GetIntermediateTransactions(t *testing.T, testContext *vm.VMTestContext) []data.TransactionHandler {
+	scForwarder := testContext.ScForwarder
+	mockIntermediate, ok := scForwarder.(*mock.IntermediateTransactionHandlerMock)
+	require.True(t, ok)
+
+	return mockIntermediate.GetIntermediateTransactions()
+}
+
+// CleaAccumulatedIntermediateTransactions -
+func CleaAccumulatedIntermediateTransactions(t *testing.T, testContext *vm.VMTestContext) {
```

Comment:
typo: Clea instead Clean ? here and in comment

Reply:
refactored.